src/app/core/hq-dashboard-sub-menu/hq-dashboard-sub-menu.component.html

```diff
@@ -8,7 +8,7 @@ <h2>HQ Dashboard</h2>
         <ul class="list-inline">
             <li class="list-inline-item"><a [routerLink]="['/last-7-day-post-list']" routerLinkActive="active">{{'last 7 days' | uppercase}}</a></li>
             <li class="list-inline-item"><a [routerLink]="['/next-7-day-activity-list']" routerLinkActive="active">{{'7 days forecast' | uppercase}}</a></li>
-            <li class="list-inline-item"><a [routerLink]="['/themes-of-the-week']" routerLinkActive="active">{{'themes of the week' | uppercase}}</a></li>
+            <li *appHasRole="['Administrators']" class="list-inline-item"><a [routerLink]="['/themes-of-the-week']" routerLinkActive="active">{{'themes of the week' | uppercase}}</a></li>
```

Comment:
only administrators are able to look at the themes of the week? Did you mean to add this to the buttons that get you to the theme-list route

Reply:
Not sure yet, but I just did this as a proof of concept for Anne and Carolynn.

Reply:
ahhh ok sorry I didn't understand that originally.